crates/collab/src/api/events.rs

```diff
@@ -1555,15 +1555,12 @@ fn for_snowflake(
             );
             map.insert("signed_in".to_string(), event.signed_in.into());
             if let Some(country_code) = country_code.as_ref() {
-                map.insert("country_code".to_string(), country_code.clone().into());
+                map.insert("country".to_string(), country_code.clone().into());
             }
         }
 
         let user_properties = Some(serde_json::json!({
             "is_staff": body.is_staff,
-            "Country": country_code.clone(),
```

Comment:
Were these telemetry changes intentional?

Reply:
Not on this branch